src/main/scala/tile/BaseTile.scala

```diff
@@ -29,6 +32,11 @@ trait TileParams {
   val name: Option[String]
 }
 
+abstract class InstantiatableTileParams[TileType <: BaseTile] extends TileParams {
```

Comment:
Can this be shortened to `Instantiable`?

Reply:
I am attempting do this but apparently i cannot spell